tools/PI/DevHome.PI/Views/BarWindowVertical.xaml.cs

```diff
@@ -59,9 +66,25 @@ private void MainPanel_Loaded(object sender, RoutedEventArgs e)
         RemoveThickFrameAttribute();
 
         // Regardless of what is set in the XAML, our initial window width is too big. Setting this to 70 (same as the XAML file)
-        Width = 70;
-    }
-
+        Width = 70;
+
+        // Calculate the DPI scale.
+        var monitor = PInvoke.MonitorFromWindow(ThisHwnd, MONITOR_FROM_FLAGS.MONITOR_DEFAULTTONEAREST);

```

Comment:
Minor nit, but I supposed this could be wrapped in a function in WindowHelpers.cs.

Reply:
Thats true, I'll make a helper GetDPIFromHwnd() or such.